lib/common/cmn_files.ml

```diff
@@ -0,0 +1,14 @@
+open Core.Std
+open Core_extended.Std
+open Async.Std
+
+exception Cmn_write_error
+
+let write root name contents =
```

Comment:
I guess you refer to https://github.com/afiniate/afin/pull/2/files ?

Call that from here then? It should be next to ready for being merged